data/scripts.js

```diff
@@ -1317,6 +1317,9 @@ exports.BattleScripts = {
 				case 'explosion':
 					if (counter.setupType || hasMove['wish']) rejected = true;
 					break;
+				case 'hiddenpower':
+					if (this.getType(move.type) === 'Grass' && hasMove['solarbeam']) rejected = true;
```

Comment:
Actually, Sunny Day / Fire Blast / Hyper Voice / HP Grass is now a real possibility and probably shouldn't be, and we should do something about that.